tests/Unit/IO/Test_VolumeData.cpp

```diff
@@ -303,3 +313,28 @@ SPECTRE_TEST_CASE("Unit.IO.H5.VolumeData.ComponentFormat1",
 #endif
   // clang-format off
 }
+
+// [[OutputRegex, No observation with value]]
+[[noreturn]] SPECTRE_TEST_CASE("Unit.IO.H5.VolumeData.FindNoObservationId",
+                               "[Unit][IO][H5]") {
+  // clang-format on
+  ASSERTION_TEST();
```

Comment:
You're testing an ERROR, not an ASSERT.

Reply:
thanks for catching this 👍 I squashed this change